openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -19,11 +21,11 @@ def __str__(self):
 
 class MultiSelectWithOtherField(MultiSelectField):
     """
-        This class is a Django Model field class that supports
-        multi select along with other option
-        The `other_max_length` parameter is required for this
-        Choice keys can not contain commas and other field can not contain
-        pipe character i.e. `|`
+    This class is a Django Model field class that supports
+    multi select along with other option
+    The `other_max_length` parameter is required for this
+    Choice keys can not contain commas and other field can not contain
```

Comment:
*cannot

https://www.grammarly.com/blog/cannot-or-can-not/#:~:text=Here%20is%20a%20quick%20summary,.%20.%20.%20but%20also.%E2%80%9D